src/profile/components/deleteProfile/DeleteProfile.tsx

```diff
@@ -149,48 +154,37 @@ function DeleteProfile(): React.ReactElement {
     dataLoadState === errorLoadState ? (
       <ServiceConnectionLoadError />
     ) : (
-      <ServiceConnectionLoadIndicator />
+      <LoadIndicator text={t('deleteProfile.loadingServices')} />
     );
 
+  if (isDeletingProfile) {
+    return (
+      <div data-testid={'deleting-profile'}>
+        <h2 ref={scrollIntoViewRef}>{t('deleteProfile.title')}</h2>
+        <LoadIndicator text={`${t('notification.removing')}...`} />
+      </div>
+    );
+  }
   return (
-    <React.Fragment>
-      <ExpandingPanel
-        title={t('deleteProfile.title')}
-        initiallyOpen={initiallyOpen}
-        scrollIntoViewOnMount={initiallyOpen}
-        onChange={onExpandingPanelChange}
-        dataTestId={'delete-profile'}
-      >
-        <p
-          dangerouslySetInnerHTML={{ __html: t('deleteProfile.explanation') }}
-        />
-        {dataLoadState !== loadedLoadState ? (
-          <LoadStateIndicator />
-        ) : (
-          <div className={styles['checkbox-container']}>
-            <Checkbox
-              onChange={handleDeleteInstructions}
-              id="deleteInstructions"
-              name="deleteInstructions"
-              checked={deleteInstructions}
-              label={t('deleteProfile.accept')}
-            />
-
-            <Button
-              type="button"
-              onClick={handleConfirmationModal}
-              disabled={!deleteInstructions}
-              className={styles.button}
-              id={removeButtonId}
-            >
-              {t('deleteProfile.delete')}
-            </Button>
-          </div>
-        )}
-      </ExpandingPanel>
-
+    <ProfileSection hasVerifiedUserData data-test-id={'delete-profile'}>
```

Comment:
I suppose the `hasVerifiedUserData` value has lost its original meaning. Now it seems to be used to mean "use `background: none` CSS style". The value and perhaps also the `.verified-data` class should probably be renamed.

or

If `hasVerifiedUserData` is `false`, then `ProfileSection` gets a white background. This seems to be used in places where the underlying background color is also white, so it doesn't add anything visible. Can the white background and `hasVerifiedUserData` flag just be removed from `ProfileSection`?

Reply:
Shouldn't the white background now be removed from the `.profile-section` style, making `ProfileSection`s transparent?

Reply:
Yes, and also useless border-radius is now also removed. 

Removed also TopSectionWithKoros component because it is not used anywhere.